tests/poly/lower_monomial.mlir

```diff
@@ -0,0 +1,16 @@
+// RUN: heir-opt --poly-to-standard %s > %t
+// RUN: FileCheck %s < %t
+
+#cycl_2048 = #poly.polynomial<1 + x**1024>
+#ring = #poly.ring<cmod=4294967296, ideal=#cycl_2048>
+
+func.func @test_monomial() {
+  // CHECK: arith.constant 1023
+  %deg = arith.constant 1023 : index
+  // CHECK: arith.constant 5
+  %five = arith.constant 5 : i32
+  // CHECK: arith.constant
+  // CHECK: tensor.insert
```

Comment:
nit: could you assert that the check inserts at index 5 for correctness?

Reply:
Done.